.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create an issue to track a bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Description
+
+A clear and concise description of what the bug is.
+
+- Does it occur on any specific browsers or breakpoints?
+- Remember to assign this to one of our GitHub project boards and to a milestone if applicable
+
+## Steps To Reproduce
+
+Steps to reproduce the behaviour:
```

Comment:
Would it make more sense here to ask for an example article? It's rarely the case that there are any 'steps' to reproduce bugs on Dotcom

Reply:
I think asking for an example page is very sensible, yes. Perhaps something like "Please include links to one or more pages where the bug can be found"? I'm saying "page" rather than "article" because it could be e.g. a front.

Regarding steps, maybe there aren't always steps beyond "go to this page and look". But sometimes maybe there's "go here, sign in, click this button", or "go here, in this browser, at this breakpoint, and scroll here". I'm thinking if there aren't specific steps then it's easy to delete the placeholder, but sometimes it's useful as a prompt?

Alternatively we could provide prompts for the environment:

- Page URL
- Browser
- Breakpoint

Or we could, as you say, just ask for an example URL and leave the rest freeform. What do you think?